src/Bookings.js

```diff
@@ -1,18 +1,36 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.jsx";
+// import fakeBookings from "./data/fakeBookings.json";
 
 const Bookings = () => {
+  const [booking, setBooking] = useState([]);
+
+  useEffect(() => {
+    fetch("https://cyf-react.glitch.me")
+      .then(res => res.json())
+      .then(data => {
+        setBooking(data);
+        // console.log(data);
+      })
+      .catch(err => console.log(err));
+  }, []);
+
   const search = searchVal => {
     console.info("TO DO!", searchVal);
+    setBooking(
+      booking.filter(
+        eachPerson =>
```

Comment:
I think we could use a better variable name here, `eachPerson` doesn't really reflect what this variable is. It is a `booking`, from our `bookings` array, so maybe we could try calling it `booking` 😄